packages/test/test-end-to-end-tests/src/test/gc/gcTombstoneDataStores.spec.ts

```diff
@@ -890,22 +862,19 @@ describeNoCompat("GC data store tombstone tests", (getTestObjectProvider) => {
 			"Can un-tombstone datastores by storing a handle",
 			[
 				// When confirming it's tombstoned
-				{ eventName: "fluid:telemetry:ContainerRuntime:GC_Tombstone_DataStore_Requested" },
-				// When reviving it
 				{
 					eventName: "fluid:telemetry:ContainerRuntime:GC_Tombstone_DataStore_Requested",
-					category: "generic",
+					clientType: "interactive",
 				},
+				// When reviving it
 				{
-					eventName:
-						"fluid:telemetry:ContainerRuntime:GarbageCollector:SweepReadyObject_Loaded",
+					eventName: "fluid:telemetry:ContainerRuntime:GC_Tombstone_DataStore_Requested",
+					category: "generic",
 				},
 				{
-					eventName:
-						"fluid:telemetry:ContainerRuntime:GarbageCollector:GC_Tombstone_Datastore_Revived",
-					category: "generic",
```

Comment:
We don't have this telemetry anymore?

Reply:
This one surprised me too. It's very hard to follow what's going on in the test. Could you take a look at this one?

Reply:
Ok so the first requested telemetry is when we get the first 404. The second one is when we re-request the tombstoned object to store it (that's why it's generic). We should still be getting a GC_tombstone_datastore_revived event. It should be generic. We should get it right before we summarize when we stored the handle of the tombstoned datastore in
`defaultDataObject._root.set("store", tombstonedObject.handle);`

Reply:
They are logged as expected but the test fails with unexpected GC_Tombstone_Datastore_Revived error. I guess it has something to do with adding generic events in itExpects. Debugging.

Reply:
Figured it out. I missed adding "GarbageCollector" to the generic event name. But the test was failing with SweepReadyObject_Revived not found so that was confusing.

Reply:
Nice, feel free to resolve this conversation